bases/lefthook/eslint-prettier/lefthook.yml

```
@@ -0,0 +1,14 @@
+# <files>の部分は対象としたいディレクトリを指定してください
+# gitでstageされたファイルのみを対象とする場合は{staged_files}を指定してください
+
+pre-commit:
+  commands:
+    fix:
+      glob: '*.{js,jsx,ts,tsx}'
```

[ask]

対象がbiome寄り少ないのはeslintが対応していないとかですかね？
parallel付けなければ順次実行できるはずなので、prettierとeslintの実行を分割してそれぞれ対象を最大化しておくと良さそう

```suggestion
      glob: '*.{js,jsx,ts,tsx}'
```

---

parallelつけたらですかね？
その場合はeslintとprettierが分割して実行できるので良いと思います！